src/interfaces/IPermit2Forwarder.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IAllowanceTransfer} from "permit2/src/interfaces/IAllowanceTransfer.sol";
+
+/// @notice Permit2Forwarder allows permitting this contract as a spender on permit2
```

Comment:
this is the exact same natspec as Permit2Forwarder. I feel like the top of interfaces should all be the same, maybe like how it is for IPositionManager.sol. 
`/// @title IPermit2Forwarder
 /// @notice Interface for the Permit2Forwarder contract`
 and then we can follow this standard across all of our interfaces, and then the natspec at the top of the contracts could go in more detail?